### PR TITLE
migrate foundation_ethereum_base_trades

### PR DESCRIPTION
**Pull request by 0xRobin**
example PR of migrating a standalone NFT model.

1. made a copy of the spark spell to `foundation_ethereum_base_trades_legacy.sql`
2. adapted all refs to use the `_legacy` spark model
3. modified the main spell to by DuneSQL compatible

checks:

- [x] dbt tests are successful
- [ ] migration data validation is successful

### CODE REVIEW

**Comment by 0xRobin** [comment]
currently blocked until we merge this:
https://github.com/duneanalytics/spellbook/pull/3676

File: macros/dune/source.sql
Comment (jkylling) [inline macros/dune/source.sql:3]: nit:
```suggestion
  {%- if target.type == 'trino' -%}
```

File: seeds/_sector/nft/_schema.yml
Comment (jkylling) [inline seeds/_sector/nft/_schema.yml:15]: Why additional columns?

**Review by jkylling — APPROVED** [review]

File: seeds/_sector/nft/_schema.yml
Comment (0xRobin) [reply seeds/_sector/nft/_schema.yml:15]: The columns where already there in the seed, but these were set as varchar by the automatic type inference, hence the new overrides to varbinary.

**Comment by jkylling** [comment]
I've ran the data quality script and verified that the data looks correct.

**Comment by jeff-dude** [comment]
note: merging with spark prod job in "dummy mode", i.e. no steps to run. trino will have the normal steps.

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**
